cmake/External/nnpack.cmake

```diff
@@ -51,6 +51,13 @@ if(ANDROID OR IOS OR ${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAM
   if(NOT DEFINED CPUINFO_SOURCE_DIR)
     set(CPUINFO_SOURCE_DIR "${CAFFE2_THIRD_PARTY_ROOT}/cpuinfo" CACHE STRING "cpuinfo source directory")
   endif()
+  # Orlando: check if avx bytecode has been generated or not
+  # also include amd.py, mmxsse.py, fma.py, amd.py, mask.py, crypto.py, generic.py
+  if(NOT EXISTS "${PYTHON_PEACHPY_SOURCE_DIR}/peachpy/x86_64/avx.py")
```

Comment:
This seems a bit questionable as if there are updates to peachpy which would have changed the contents of avx.py, those updates won't be taken unless you do a clean. It is even worse than that, because a shallow clean in only our top level repo won't update avx.py; you have to do a deep clean in the submodules as well. Unfortunately, I'm not sure what a better way to do this is.

Reply:
@ezyang yes, let's me check if peachpy has any functionality to check if instruction code has already been generated or not. or we can let it easy, just cleanup previous generated instruction and regenerate again.

Reply:
@ezyang now I use python setup.py develop --uninstall to remove installation first, then python setup.py develop to reinstall so if avx.py or other generated stuffs exists, they will be overwritten. ^-^

Reply:
This is why I'm not sure what the better way to do this is; I also don't want to be reinstalling peachpy every time I run my build script 🤣